test/e2e/framework/pod/utils.go

```diff
@@ -294,19 +294,24 @@ func FindContainerStatusInPod(pod *v1.Pod, containerName string) *v1.ContainerSt
 // VerifyCgroupValue verifies that the given cgroup path has the expected value in
 // the specified container of the pod. It execs into the container to retrive the
 // cgroup value and compares it against the expected value.
-func VerifyCgroupValue(f *framework.Framework, pod *v1.Pod, cName, cgPath, expectedCgValue string) error {
+func VerifyCgroupValue(f *framework.Framework, pod *v1.Pod, cName, cgPath string, expectedCgValues ...string) error {
 	cmd := fmt.Sprintf("head -n 1 %s", cgPath)
-	framework.Logf("Namespace %s Pod %s Container %s - looking for cgroup value %s in path %s",
-		pod.Namespace, pod.Name, cName, expectedCgValue, cgPath)
+	framework.Logf("Namespace %s Pod %s Container %s - looking for one of the expected cgroup values %s in path %s",
+		pod.Namespace, pod.Name, cName, expectedCgValues, cgPath)
 	cgValue, _, err := ExecCommandInContainerWithFullOutput(f, pod.Name, cName, "/bin/sh", "-c", cmd)
 	if err != nil {
-		return fmt.Errorf("failed to find expected value %q in container cgroup %q", expectedCgValue, cgPath)
+		return fmt.Errorf("failed to find one of the expected cgroup values %q in container cgroup %q", expectedCgValues, cgPath)
 	}
 	cgValue = strings.Trim(cgValue, "\n")
-	if cgValue != expectedCgValue {
-		return fmt.Errorf("cgroup value %q not equal to expected %q", cgValue, expectedCgValue)
+
+	for _, expectedCgValue := range expectedCgValues {
+		if cgValue == expectedCgValue {
+			return nil
+		}
 	}
-	return nil
+
+	return fmt.Errorf("namespace %s pod %s container %s - cgroup value %q in path %q not equal to one of the expected cgroup values %q", pod.Namespace, pod.Name, cName, cgValue, cgPath, expectedCgValues)
```

Comment:
Maybe instead:

```go
return framework.Gomega().Expect(cgValue).To(gomega.BeElementOf(expectedCgValues),
	fmt.Sprintf("value of cgroup %q for container %q should match one of the expectations", cgPath, cName))
```

(no need to include namespace/pod, it will be included when the error is reported)

Reply:
I updated this to be _close_ to what you suggested... feel free to let me know if I misunderstood